### PR TITLE
chore(deps): bump dependencies version

### PR DESCRIPTION
**Pull request by r17x**
Ref: https://github.com/zainfathoni/obel/issues/216

@zainfathoni After `bump version` of `react-scripts`, we got an error when start or build this project cause `eslint` & relate dependencies it's `conflict` with `react-scripts`.

I think we can solve with this options:
- [x] 1. REMOVED `eslint` & conflict dependencies with `react-scripts` (https://github.com/zainfathoni/obel/pull/217#issuecomment-619459613)
- ~~2. `SKIP_PREFLIGHT_CHECK` in `.env` file.~~ (Hard debug for it)

WDYT ? 
   
image
   
image

### CODE REVIEW

**Comment by zainfathoni** [comment]
@ri7nz I agree with the 1st option, because the 2nd one potentially leads to hard-to-debug issues in the future.

However, `prettier-eslint-cli` also depends on it.
image

So we have to remove it while maintaining the ability to format the code using `yarn format` command. Please try to find another solution for this. I think we should be able to manually configure `prettier` only (without `prettier-eslint-cli`) and `eslint` separately, then run them consecutively.

Thank you. 🙏

**Comment by andriawan** [comment]
should we really need to upgrade react-scripts @ri7nz now? i try to run locally and it was fine...

**Comment by zainfathoni** [comment]
Nice insight, yeah, we can actually afford to defer upgrading `react-scripts` for now, as long as the security warnings from GitHub are resolved correctly.

**Comment by ri7nz** [comment]
@zainfathoni 

finally, I'm just remove `babel-eslint` and make match version (eslint in project same with eslint in react-scripts). 

So, we can run development normally (`yarn start`).

File: package.json [hunk 1]
```diff
@@ -42,11 +42,10 @@
     "@testing-library/cypress": "5.0.1",
     "@testing-library/jest-dom": "4.2.0",
     "@testing-library/react": "9.3.2",
-    "autoprefixer": "9.6.4",
-    "babel-eslint": "10.0.2",
+    "autoprefixer": "9.7.5",
     "cross-env": "6.0.3",
     "cypress": "3.4.1",
-    "eslint": "6.5.1",
+    "eslint": "6.8.0",
```
Comment (ri7nz) [inline package.json:48]: just make it match with version `eslint` in `react-scripts` (use: npm ls eslint or yarn list eslint).

**Review by zainfathoni — APPROVED** [review]
LGTM 💯

**Comment by r17x** [comment]
Thank you @zainfathoni

**Comment by zainfathoni** [comment]
You're welcome @ri7nz , thank you for finding the simplest solution. 😁 🙏

**Pull request merged**